plugin/storage/cassandra/schema/docker.sh

```diff
@@ -3,7 +3,7 @@
 # This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
 # that allows installing Jaeger keyspace and schema without installing cqlsh.
 
-CQLSH=/opt/apache-cassandra-3.0.12/bin/cqlsh
+CQLSH=/usr/bin/cqlsh
```

Comment:
Could we make this configurable? And leave other env variables to be set to default values in `create.sh`?